client/my-sites/checkout/composite-checkout/wpcom/components/wp-order-review-line-items.js

```diff
@@ -130,44 +131,35 @@ WPLineItem.propTypes = {
 	onChangePlanLength: PropTypes.func,
 };
 
-function LineItemTitle( { item, id } ) {
-	if ( isLineItemADomain( item ) ) {
-		return <LineItemDomainTitle item={ item } id={ id } />;
-	}
-	if ( item.type === 'domain_map' || item.type === 'offsite_redirect' ) {
-		return <LineItemDomainTitle item={ item } id={ id } />;
-	}
+function LineItemMessage( { item } ) {
+	const translate = useTranslate();
 	return (
-		<LineItemTitleUI>
-			<ProductTitleUI id={ id }>{ item.label }</ProductTitleUI>
-		</LineItemTitleUI>
+		<LineItemMessageUI>
+			{ 'plan' === item.type && translate( 'Billed yearly' ) }
+			{ item.wpcom_meta?.is_bundled && item.amount.value === 0 && (
+				<BundledDomainFreeUI>{ translate( 'First year free' ) }</BundledDomainFreeUI>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'First year free with your plan' )` *ES Score: 6*